edweiss-app/__test__/memento/card.test.tsx

```diff
@@ -134,26 +138,28 @@ describe('CardListScreen', () => {
 	});
 
 	it('renders without crashing', () => {
-		const { getByText } = render(<CardListScreen />);
-		expect(getByText('Question 1')).toBeTruthy();
-		expect(getByText('Question 2')).toBeTruthy();
+		const { getByTestId } = render(<CardListScreen />);
+		expect(getByTestId('cardQuestionIndex_0')).toBeTruthy();
 	});
 
 	it('toggles card selection', () => {
-		const { getByText } = render(<CardListScreen />);
-		const question1 = getByText('Question 1');
-		const question2 = getByText('Question 2');
+		const { getByText, getByTestId } = render(<CardListScreen />);
+		const question1 = getByTestId('cardQuestionIndex_0');
+		const question2 = getByTestId('cardQuestionIndex_1');
 
 		fireEvent(question1, 'onLongPress');
```

Comment:
I did not take a look yet but I prefer ask : does the action of long press is intuitive enough for the user? If not sure, did you manage to make sure the user knows that he could do this? (an explicative text or anything else)

Reply:
Thank you for the insight! I haven’t added any instructions for users yet, but I’ll definitely keep this in mind. I plan to implement a set of clear instructions for new users to ensure they understand how to use this feature.